examples/mobile/HomeApp/css/style.css

```diff
@@ -34,15 +34,80 @@
 .app-movie-poster-next {
     width: 50%;
 }
+
+.app-remote-control {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    text-align: center;
+    padding: 5px 0;
+}
+.app-volume-control {
+    display: flex;
+    flex-direction: column;
+    border-radius: 20px;
+    box-shadow: 1px 1px 3px 0 var(--overlay);
+    margin: 5px;
+    font-size: 16px;
+    background-image: linear-gradient(170deg,
+        rgba(0, 0, 0, 0.2) 0%, var(--background-area-color) 15%,
+        var(--background-area-color) 85%, rgba(0, 0, 0, 0.2) 100%);
+}
+.app-chanel-control {
+    display: flex;
+    flex-direction: column;
+    border-radius: 20px;
+    box-shadow: 1px 1px 3px 0 var(--overlay);
+    margin: 5px;
+    font-size: 16px;
+    background-image: linear-gradient(170deg,
+        rgba(0, 0, 0, 0.2) 0%, var(--background-area-color) 15%,
+        var(--background-area-color) 85%, rgba(0, 0, 0, 0.2) 100%);
+}
+.app-four-way-direction {
```

Comment:
four-way => 4way.

I think that it looks good.

Reply:
Done